_templates/exchanges.html

```diff
@@ -144,6 +146,8 @@ <h2 class="exchanges-tab-title exchanges-europe-title" id="europe">Europe</h2>
       <p>
         <a class="marketplace-link" href="https://anycoindirect.eu">AnyCoin Direct</a>
         <br>
+        <a class="marketplace-link" href="https://www.binance.je/">Binance Jersey</a>
```

Comment:
Since Jersey is part of the United Kingdom; we recommend this particular entry be placed in that corresponding section.

Reply:
Okay. I will submit the changes.